### ISSUE TITLE
create index now adds illegal parantheses around columns

### ISSUE THREAD

**Issue by narrowtux** [elixir-ecto/ecto#2257]
Before 2.1.6,

```elixir
create index(:readings, ["\"measured_at\" DESC", "\"location\" NULLS LAST"])
```

generated this SQL query:

```sql
CREATE INDEX "readings_measured_at_DESC_location_NULLS_LAST_index" ON "readings" ("measured_at" DESC, "location" NULLS LAST);
```

now it generates this one: (added parantheses around each column, which are illegal syntax)

```sql
CREATE INDEX "readings_measured_at_DESC_location_NULLS_LAST_index" ON "readings" (("measured_at" DESC), ("location" NULLS LAST));
```

This might have been done to make it easier to add expression to indexes, however they are not the only things we need the free-form input for. Now postgres considers everything in the parantheses as an expression and some of the old migrations have failed.

**Comment by josevalim** [comment]
This has been fixed on Ecto v2.2.0 and later: https://github.com/elixir-ecto/ecto/commit/c093edaebc87fdb50cdf5e14422fa22cf6097c79

**Comment by narrowtux** [comment]
Can't upgrade to ecto 2.2.0 just yet, so I downgraded to 2.1.4
